components/Common/Button/index.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import Button from './';
+import classNames from 'classnames';
+
+type Story = StoryObj<typeof Button>;
+type Meta = MetaObj<typeof Button>;
+
+export const Default: Story = {
+  args: {
+    variant: 'primary',
+    children: 'Download Node (LTS)',
+    disabled: false,
+  },
+  decorators: [
+    (Story, { args }) => (
+      <>
+        <div
+          className={classNames({
+            'bg-[#000000] w-fit rounded-md': args.variant === 'special',
```

Comment:
Can we make the whole container black? Just to see border effect looking correct?

Reply:
Nvm, I'm adding this here: https://github.com/nodejs/nodejs.org/pull/5875

Reply:
Can we remove then this className part for the special button?